server/app/models/track.js

```diff
@@ -27,13 +16,28 @@ var TrackSchema = new Schema({
     ref: 'User',
     required: true
   },
-  location: locSchema,
+  places: [{
+    type: Schema.Types.ObjectId,
+    ref: 'Place',
+    required: true
+  }],
   description: String,
   trackRate: Number,
```

Comment:
trackRate should count by number of likes. But I think, it'll be in future. Now it is good for me :)